python/ray/util/placement_group.py

```diff
@@ -83,6 +83,19 @@ def ready(self) -> ObjectRef:
             placement_group_bundle_index=bundle_index,
             resources=resources).remote(self)
 
+    def wait(self, timeout_ms: int) -> bool:
```

Comment:
Can we use second as timeout measure? I think that's consistent with other ray APIs

Reply:
I have fix it, thanks.